js/base/functions/platform.js

```diff
@@ -1,4 +1,3 @@
-"use strict";
```

Comment:
is it possible to keep the "use stict" in the headers?

Reply:
@frosty00 esm modules are strict by definition 😛